android/src/main/java/com/swmansion/gesturehandler/react/RNGestureHandlerButtonViewManager.kt

```diff
@@ -291,6 +291,15 @@ class RNGestureHandlerButtonViewManager : ViewGroupManager<ButtonViewGroup>(), R
       val eventTime = event.eventTime
       val action = event.action
 
+      if (touchResponder != null && touchResponder !== this) {
```

Comment:
Wouldn't this be enough?

```suggestion
      if (touchResponder !== this) {
```

Reply:
Using only if (touchResponder !== this) would also be true when touchResponder is null, so when no other button has grabbed the responder. In that case, we want to proceed normally.

Reply:
Oh, you're right 😅